recipes/itsxpress/meta.yaml

```diff
@@ -22,14 +22,14 @@ build:
 requirements:
   host:
     - pip
-    - python >=3.8
+    - python =3.8.16
```

Comment:
pinning  a recipe this strict is not a good idea. Why is python 3.9 not supported? or 3.10?

Reply:
I agree, this is unnecessary. I'll change it back to >=3.8 shortly. This was originally to help with dependency issues with Qiime2, as this package can also be used as a plugin for Qiime but those issues have been fixed.